### PR TITLE
Adding additional service display tests

### PR DESCRIPTION
**Pull request by PhysicsJon**
[AB#8133](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/8133)
[AB#5123](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/5123)

### CODE REVIEW

File: src/OrderFormAcceptanceTests.Steps/Steps/CatalogueSolutions.cs [hunk 1]
```diff
@@ -45,6 +45,14 @@ public void ThenTheUserIsAbleToManageTheCatalogueSolutionsSection()
             ThenTheCatalogueSolutionDashboardIsPresented();
         }
 
+        [Then(@"the User is able to manage the Additional Services section")]
+        public void ThenTheUserIsAbleToManageTheAdditionalServicesSection()
+        {
+            Test.Pages.OrderForm.ClickEditAdditionalServices();
+            
+        }
+
```
Comment (talawson05) [inline src/OrderFormAcceptanceTests.Steps/Steps/CatalogueSolutions.cs:54]: can you move this step into AdditionalServices

Comment (PhysicsJon) [reply src/OrderFormAcceptanceTests.Steps/Steps/CatalogueSolutions.cs]: done

File: src/OrderFormAcceptanceTests.Tests/Features/ManageOrderForm/AdditionalServices.feature [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+And there is the Supplier section
+And there is the Commencement date section
+And there is the Service Recipients section
+And there is the Additional Service section
+And the Preview order summary button is enabled
+And the Delete order button is enabled
+And the Submit order button is disabled
+
+Scenario: Additional Services - Catalogue Solution now complete, >= 1 Service Recipient, 0 Catalogue Solution
+Given an unsubmitted order exists
```
Comment (talawson05) [inline src/OrderFormAcceptanceTests.Tests/Features/ManageOrderForm/AdditionalServices.feature:27]: can you move this data setup into a background please?

Comment (PhysicsJon) [reply src/OrderFormAcceptanceTests.Tests/Features/ManageOrderForm/AdditionalServices.feature]: Done

**Pull request merged**